### PR TITLE
Document the IO class

### PR DESCRIPTION
**Pull request by sbrl**
Following up from issue #274 that I opened, I have attempted to document the IO class based on the io.wren file that @bjorn showed me. There are ~~probably~~ lots of mistakes in this documentation I have written.

Feel free to ignore this pull request.

### CODE REVIEW

File: doc/site/core/IO.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
+^title IO Class
+^category core
+
+The IO class can be used to read and write to and from the console.
+
+## Static Methods
+
+### IO.print(obj[, obj2, obj3...])
+Prints any number of things to the console and then prints a new line character. If you don't pass it a string, it will be converted to a string for you. When passed multiple things, Wren outputs one after another.
```
Comment (munificent) [inline doc/site/core/IO.markdown]: Two style nits, but can you add a blank line after the header and hard wrap this paragraph to fit in 80 columns? (Hard wrapping makes diffs and code reviews easier to read.)

File: doc/site/core/IO.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+
+	> IO.print("I like bananas")
+	I like bananas
+	> IO.print("Oranges", 10)
+	Oranges10
+	> 
+
+### IO.write(obj)
+Prints a single thing to the console, but does not print a new line character afterwards. If you pass it something that isn't a string, it will convert it to a string.
```
Comment (munificent) [inline doc/site/core/IO.markdown]: "newline" should be one word.

File: doc/site/core/IO.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
+^title IO Class
```
Comment (munificent) [inline doc/site/core/IO.markdown]: Do you mind making the file name lowercase ("io.html") to match the other files?

File: doc/site/core/index.markdown [hunk 4]
```diff
@@ -20,5 +20,6 @@ All Wren source files automatically have access to the following classes:
 * [Range](range.html)
 * [Sequence](sequence.html)
 * [String](string.html)
+* [IO](IO.html)
```
Comment (munificent) [inline doc/site/core/index.markdown]: And then make the URLs lowercase here too.

**Comment by munificent** [comment]
Sorry for the very long delay, but this looks awesome! I had a few minor comments, but I'll be delighted to merge this in if you don't mind tweaking it a bit first.

Thank you!

**Comment by sbrl** [comment]
I have added another commit to this pull request fixing those issues. Does that fix it?

File: doc/site/core/IO.markdown [hunk 5]
```diff
@@ -0,0 +1,44 @@
+^title io Class
```
Comment (bjorn) [inline doc/site/core/IO.markdown]: I'm pretty sure the lowercasing was just about the filenames, not this title. :-)

Comment (munificent) [inline doc/site/core/IO.markdown]: Yes, @bjorn is right. Sorry I wasn't clear!

**Comment by sbrl** [comment]
Title fixed!

File: doc/site/core/IO.markdown (hunk 5 above)
Comment (sbrl) [inline doc/site/core/IO.markdown]: No problem :)

**Comment by munificent** [comment]
This is perfect, thank you!

**Comment by munificent** [comment]
Oh, and feel free to send me a PR to add yourself to the AUTHORS file if you'd like. :D

**Comment by sbrl** [comment]
Submitted PR #281!

**Pull request merged**